### PR TITLE
add icons and animations on hover over tiles

### PR DESCRIPTION
**Pull request by andpodob**
Hi,
I finished creating tile view, also added icons on tiles and animation. Below I'm placing img of how it look now. I encourage you to run app to look how animations work.
final_tile_view

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,22 +4,54 @@
...
+           <div class="tile" id="tile3">
+               <div class="icon-container"> 
+                <i class="material-icons md-48">chat</i>
+                <p>Contact</p>
+               </div>
+           </div>
+           <div class="tile" id="tile4">
+            <div class="icon-container">    
+                <img src="images/GitHub-Mark-64px.png " alt="github icon" width="48px" height="48px"> 
+                <p>My Github account</p>
```
Comment (yongli-abc) [inline portfolio/src/main/webapp/index.html:56]: This text wraps to a second line when I hover over it. Try to see if you can fix that :)

**Review by yongli-abc — APPROVED** [review]
I managed to run the app on my own \o/ The hovering animation looks cool!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (andpodob) [reply portfolio/src/main/webapp/index.html:56]: Sure, I will work on paragraph styles and pop-up actions today, thanks for advice.

Comment (yongli-abc) [reply portfolio/src/main/webapp/index.html:56]: Sounds good. Feel free to merge the PR.

**Pull request merged**